imageregistry/v1/00-crd.yaml

```diff
@@ -0,0 +1,555 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: configs.imageregistry.operator.openshift.io
+spec:
+  group: imageregistry.operator.openshift.io
+  scope: Cluster
+  version: v1
+  names:
+    kind: Config
+    listKind: ConfigList
+    plural: configs
+    singular: config
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: Config is the configuration object for a registry instance managed
+        by the registry operator
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
```

Comment:
@ricardomaraschini you need to re-run `make update` so that the crd yaml is updated with the lower case version of all these different field names

Reply:
Thanks for catching it!